packages/mdc-textfield/icon/_mixins.scss

```diff
@@ -42,58 +52,56 @@
 ///
 @mixin mdc-text-field-disabled-icon-color($color) {
   &.mdc-text-field--disabled {
-    @include mdc-text-field-icon-color_($color);
+    @include mdc-text-field-leading-icon-color_($color);
+    @include mdc-text-field-trailing-icon-color_($color);
   }
 }
 
 // Private mixins
 
-@mixin mdc-text-field-icon-horizontal-position_($position-property, $position, $padding, $input-padding) {
-  .mdc-text-field__icon {
-    @include mdc-rtl-reflexive-position($position-property, $position);
-  }
-
-  // Move the input's position, to allow room for the icon
+@mixin mdc-text-field-icon-horizontal-position_($position-property, $position) {
```

Comment:
This looks great! Let's simplify it more by creating two separate mxins, `mdc-text-field-leading-icon-horizontal-position_` and `mdc-text-field-trailing-icon-horizontal-position_` and using those mixins directly. It'll make it clearer what each mixin is doing.

Reply:
@asyncLiz This should be the last outstanding fix.

Reply:
Woops, missed this, thanks for the reminder